frontend-nextjs/Dockerfile.frontend-nextjs.new

```
@@ -0,0 +1,50 @@
+ENV NEXT_PUBLIC_ALLOWED_EMAIL_DOMAINS=$NEXT_PUBLIC_ALLOWED_EMAIL_DOMAINS
+ARG NEXT_PUBLIC_BACKEND_HOST
+ENV NEXT_PUBLIC_BACKEND_HOST=$NEXT_PUBLIC_BACKEND_HOST
+
+RUN apt-get update && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /frontend-nextjs
+
+COPY package.json pnpm-lock.yaml ./
+
+RUN npm install -g pnpm && pnpm install --frozen-lockfile
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)